src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -26,6 +26,10 @@ public partial class FrameworkElement : IEnumerable
 
 		partial void OnLoadingPartial();
 
+		internal virtual void OnPostLoading()
```

Comment:
should this be `private protected`? Also why not just declare it once in the shared `FrameworkElement.cs` file?